packages/customize-widgets/src/components/sidebar-block-editor/index.js

```diff
@@ -28,33 +35,64 @@ export default function SidebarBlockEditor( { sidebar } ) {
 		modal: false,
 		animated: 150,
 	} );
+	const [ isInspectorOpened, setIsInspectorOpened ] = useState( false );
 	const settings = useMemo(
 		() => ( {
 			__experimentalSetIsInserterOpened: inserter.setVisible,
 		} ),
 		[ inserter.setVisible ]
 	);
+	const collapse = () => setIsInspectorOpened( false );
```

Comment:
I chose the name because it's the one used in core: https://core.trac.wordpress.org/browser/trunk/src/js/_enqueues/wp/customize/controls.js#L1253

We don't have to follow their names here though, we can rename this 👍.